invenio_rdm_records/cli.py

```diff
@@ -115,22 +119,25 @@ def create_fake_record():
             "publication_date": fake_edtf_level_0(),
             "subjects": [{
                 "subject": fake.word(),
-                "identifier": "subj-1",
-                "scheme": "no-scheme"
+                "identifier": "03yrm5c26",
+                "scheme": "ror"
             }, {
                 "subject": fake.word(),
-                "identifier": "subj-1",
-                "scheme": "no-scheme"
+                "identifier": "03yrm5c26",
+                "scheme": "ror"
             }],
             "contributors": [{
-                "family_name": fake.last_name(),
-                "given_name": fake.first_name(),
-                "type": "personal",
+                "person_or_org": {
+                    "family_name": fake.last_name(),
+                    "given_name": fake.first_name(),
+                    "type": "personal",
+                },
                 "affiliations": [{
                     "name": fake.company(),
-                    "identifiers": {
-                        "ror": "03yrm5c26"
-                    }
+                    "identifiers": [{
+                        "scheme": "ror",
+                        "identifier": "03yrm5c26",
+                    }]
                 }],
                 "role": "rightsholder"
             } for i in range(3)],
```

Comment:
Until https://github.com/inveniosoftware/invenio-drafts-resources/pull/88/files#diff-839975cf50e15f8d6cbae751b63f629f75adbf5b9c1c6721d62b352d0ea6400eR23 is merged you will need to comment out the `dates` field when you create demo data!